lib/systemd_testsuite_test.pm

```diff
@@ -20,17 +20,26 @@ use known_bugs;
 use testapi;
 use power_action_utils 'power_action';
 use utils 'zypper_call';
-use version_utils;
+use version_utils qw(is_opensuse is_sle is_tumbleweed);
 
 
 sub testsuiteinstall {
     my $qa_head_repo = get_var('QA_HEAD_REPO', '');
     if (!$qa_head_repo) {
-        if (is_leap('15.0+')) {
-            $qa_head_repo = 'https://download.opensuse.org/repositories/devel:/openSUSE:/QA:/Leap:/15/openSUSE_Leap_15.0/';
+        if (is_opensuse()) {
+            my $sub_project;
+            if (is_tumbleweed()) {
+                $sub_project = 'Tumbleweed/openSUSE_Tumbleweed/';
+            }
+            else {
+                (my $version, my $service_pack) = split('\.', get_required_var('VERSION'));
+                $sub_project = "Leap:/$version/openSUSE_Leap_$version.$service_pack/";
+            }
+            $qa_head_repo = 'https://download.opensuse.org/repositories/devel:/openSUSE:/QA:/' . $sub_project;
         }
-        elsif (is_sle('15+')) {
-            $qa_head_repo = 'http://download.suse.de/ibs/QA:/SLE15/standard/';
+        else {
+            (my $version, my $service_pack) = split('-', get_required_var('VERSION'));
```

Comment:
Just a note: this might throw undef for `$service_pack` but I'm not 100% sure, I guess we'll see with SLE16 :)

Reply:
My verification runs covers when GM (SLE15, Leap15) and also service packs (SLE15-SP1, Leap15.1), so, it should also work for SLE16.

Reply:
Yes, because it is required that the variable exists, but not the isotovideo SETTING.